### DISCUSSION TITLE
hive catalog 查询报错Unknown database 'default_cluster:test_db'

### DISCUSSION THREAD

**Discussion by mengbaba3316** [apache/doris#23077] [A - General / Q&A]
hive 3.1.2
doris 1.2.2
test_db是测试的库
创建hive catalog的语句是 
`CREATE CATALOG hadoop01_hive PROPERTIES (
    'type'='hms',
    'hive.metastore.uris' = 'thrift://192.168.150.86:9083',
    'hadoop.username' = 'root'
);`
切换catalog 到 hive  查询 下面的数据库都没有问题，use 数据库 和查询表，就会提示 Unknown database 'default_cluster:test_db'
这是啥原因呀 ？

**Comment by LemonLiTree** [comment]
Upgrade to 1.2.6 to see

**Reply by mengbaba3316** [reply]
Thank you for your reply, but now I have upgraded to 1.2.6 and there is a new problem. I still create catalog normally, but the error message is`[HY000][1105] HMSClientException, msg: org.datanucleus.exceptions.NucleusUserException: Error : Could not find API definition for name "JDO". Perhaps you dont have the requisite datanucleus-api-XXX jar in the CLASSPATH?`
